assert/assertions_test.go

```diff
@@ -555,7 +555,7 @@ func TestNoError(t *testing.T) {
 	}()
 
 	if err == nil { // err is not nil here!
-		t.Errorf("Error should be nil due to empty interface", err)
+		t.Errorf("Error should be nil due to empty interface: %s", err)
```

Comment:
Yep, this will work but the `%s` format specified is not intended to be used for arbitrary types:

From https://golang.org/pkg/fmt/:
```
%s	the uninterpreted bytes of the string or slice
```

Whereas we really want `%v`:
```
%v	the value in a default format
```